mkdocs.yml

```diff
@@ -8,23 +8,22 @@ nav:
   - Home: index.md
   - Introduction:
       - background.md
-      - migration_guide.md
       - Schema: nextflow_schema/index.md
+      - nextflow_schema/nextflow_schema_examples.md
+      - nextflow_schema/sample_sheet_schema_examples.md
+  - Tutorials:
       - nextflow_schema/create_schema.md
+      - migration_guide.md
```

Comment:
Not sure the migration guide is a tutorial. We made the decision to set it at the front to make sure it was easily found by people doing the migration and the migration is still going on so not sure if it's a good idea to move it yet?

Reply:
A guide is a tutorial on how to do something. And the name of the page hasn't changed, just the section in the sidebar. Also it's only moved down two lines too, so I think it's not too different to scare people.

It certainly is not a 'backround' or 'intro' to the entire purpose of the plugin

Reply:
Okay fair enough :p